dubbo-filter/dubbo-filter-cache/src/test/java/com/alibaba/dubbo/cache/filter/CacheFilterTest.java

```diff
@@ -16,78 +16,47 @@
  */
 package com.alibaba.dubbo.cache.filter;
```

Comment:
The change made on UT doesn't look right. The original UT can test against all cache types.

Reply:
you should rollback your change made against CacheFilterTest. This test uses `@RunWith(Parameterized.class)` to test against `lru`, `jcache`, `threadlocal`.

```java
    @Parameters		
     public static List<Object[]> cacheFactories() {		
         return Arrays.asList(new Object[][]{		
                 {"lru", new LruCacheFactory()},		
                 {"jcache", new JCacheFactory()},		
                 {"threadlocal", new ThreadLocalCacheFactory()}		
         });		
     }
```

you should consider how to add `expiring` into this test, or write a new test delegated to `expiring` only.